### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by sadaf-Daneshgar**
New Features:
✅ Add desktop version
✅ Add menu bar functionality.
✅ Add functionality for responsive page.
✅ Configure linters.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+}
+
+.items {
+  padding-left: 20px;
+  margin-right: 50px;
+}
+
+.items h1 {
+  font-size: 2em;
+}
```
Comment (ABDULALI3468) [inline style.css:128]: Please consider using the same margin-top used in the Figma Design to make it closer to the design. On the desktop version 💻  there needs to be a `margin-top of 48px`.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+  font-size: 17px;
+  line-height: 24px;
+  transition: ease-in 0.4s all;
+}
+
+.last-btn:hover {
+  background: #4053fc;
+  color: #fff;
+  cursor: pointer;
+}
```
Comment (ABDULALI3468) [inline style.css]: Please consider matching your hover background color as used in the FIGMA design to make it closer to the design. 🙇 

### CODE DEMONSTRATION:-

```
.last-btn:hover {
  background: #6070FF;
  color: #fff;
  cursor: pointer;
}
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: #fff;
+  border: none;
+  color: #6070ff;
+  text-decoration: none;
+  border-radius: 10px;
+  margin-right: -20px;
+}
```
Comment (ABDULALI3468) [inline style.css:389]: Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file.

## Solution:--

```
.btn:hover{
    background-color: #6070FF;
    color: white;
}
```

```
.btn:active{
    background-color: #2230D2;
    color: white;
}
```

## Demonstration:--

image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,593 @@
...
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 20px;
+  margin-top: 10px;
+  padding: 0 20px;
+}
```
Comment (ABDULALI3468) [inline style.css]: Please consider using the same `color and width properties `used in the Figma Design to make it closer to the design. 

```
.section-4-container p {
  color: #EBEBFF;
  width: CHECK DESIGN
}
```

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @sadaf-Daneshgar 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯

### REMEMBER:-

IT WAS A SOLO ACTIVITY AND I CAN SEE COMMITS FROM OTHER PEOPLE OTHER THAN YOU. It's ok if you take help or advice from someone other ethically but seeing commits from other people makes us feel you were cheating. Please consider its for your benefit and consider these instructions for the next time ☑️ 


## Highlights
☑️Well implemented design.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (sadaf-Daneshgar) [reply style.css:128]: kindly it means I should add this code in media part and I should put margin-top: 48px, am I right?

**Comment by sadaf-Daneshgar** [comment]
Add the margin-top.
Change color of buttons for hover.
Change the width

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @sadaf-Daneshgar ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No Linters Errors :100: 
- Good Summary :+1: 
- Good PR Description :1st_place_medal: 

## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RileyManda — COMMENTED** [review]
Hi @sadaf-Daneshgar  🙋‍♀️

Great job implementing the changes suggested by the previous reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Good branch name. Great job!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_**Please see the comments under review**_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chaw-Ma-Ma-Win** [comment]
✅ Styling is looking great, perfect!
✅ Great work using the right GitHub Flow
✅ Good branch name. Great job!
✅ Linters are working properly, great job!

File: index.html [hunk 5]
```diff
@@ -1,12 +1,337 @@
...
+                    <li>Languages</li>
+                    <i class="fa fa-angle-down" aria-hidden="true"></i>
+                </ul>
+                <div class="wrapper">
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1.png" />
+                        <h5 class="m-l">javascript</h5>
+                    </div>
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1 (1).png" />
```
Comment (RileyManda) [inline index.html:282]: [Optional] but recommended for professional purposes.
I would recommend renaming your images and assets to use small caps and remove spaces between the image names for SEO purposes as well semantics.

File: index.html [hunk 6]
```diff
@@ -1,12 +1,337 @@
...
+                <ul>
+                    <li>
+                        <label for="name">
+                            <input class="full-name" type="text" placeholder="Lorem ipsum" id="name" maxlength="30"
+                                required name="fullname">
+                        </label>
+                    </li>
+                    <li>
+                        <label for="email">
+                            <input type="email" placeholder="loremipsum@gmail.com" id="email" maxlength="30" required
```
Comment (RileyManda) [inline index.html:316]: [Optional]Perhaps changing the email placeholder from `placeholder="loremipsum@gmail.com" ` would make it more appealing and easy to understand to the user.Something more user friendly like "youremail@gmail.com" Would look more appealing.

**Review by RileyManda — COMMENTED** [review]
Hi @sadaf-Daneshgar  🙋‍♀️

Great job implementing the changes suggested by the previous reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Good branch name. Great job!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_**Please see the comments under review**_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**